web/modules/custom/first_custom_module/first_custom_module.links.menu.yml

```diff
@@ -0,0 +1,19 @@
+first_custom_module.Page1:
+  title: 'Introduction'
+  description: 'This is the introduction page'
+  route_name: first_custom_module.Page1
+  weight: 7 
+  menu_name: main
+  options:
+    attributes:
+      target: _blank
+
+first_custom_module.Page2:
+  title: 'Introduction-2'
+  description: 'This is the second intro page'
+  route_name: 'first_custom_module.Page2' 
+  weight: 100 
+  menu_name: 'main'
+  options:
+    attributes:
+      target: '_blank'
```

Comment:
This red circle indicates that the new line at end of file is missing.
Did you run the code through the plugin I mentioned in slack?